maintainers/maintainer-list.nix

```
@@ -4520,6 +4520,11 @@
     github = "minijackson";
     name = "Rémi Nicole";
   };
+  mir06 = {
+    email = "armin.leuprecht@uni-graz.at";
+    githup = "mir06";
+    name = "Armin Leuprecht
```

Fix ofborg eval error:

```suggestion
    name = "Armin Leuprecht";
```

---

Ups.